templates/Containerisation/helm/helm-install.yaml

```diff
@@ -0,0 +1,112 @@
+parameters:
+    - name: prNumber
+      type: string
+    - name: privateKeyFileName
+      type: string
+    - name: appId
+      type: string
+    - name: envValuesTemplatePath
+      type: string
+    - name: envValuesPath
+      type: string
+    - name: acrName
+      type: string
+    - name: repositoryName
+      type: string
+    - name: prTagName
+      type: string
+    - name: dnsRecordName
+      type: string
+    - name: dnsZone
+      type: string
+    - name: devEnv
+      type: string
+      default: ""
+    - name: chartName
+      type: string
+    - name: chartPath
+      type: string
+    - name: subscriptionName
+      type: string
+    - name: aksSubscriptionName
+      type: string
+    - name: aksResourceGroupName
+      type: string
+    - name: aksClusterName
+      type: string
+    - name: dnsSubscriptionName
+      type: string
+    - name: dnsResourceGroupName
+      type: string
+    - name: zoneName
+      type: string
+    - name: recordName
+      type: string
+    - name: clusterResourceGroup
```

Comment:
Is this the same as 'aksResourceGroupName'

Reply:
good catch!